src/output-json.c

```diff
@@ -734,6 +734,7 @@ static int CreateJSONEther(
         if (PacketIsEthernet(p)) {
             const EthernetHdr *ethh = PacketGetEthernet(p);
             jb_open_object(js, "ether");
+            jb_set_uint(js, "ether_type", ethh->eth_type);
```

Comment:
can you create a ticket for a keyword for this? Guess it would become `ether.type`? Or `ethernet.type` more likely.

Reply:
https://redmine.openinfosecfoundation.org/issues/7531